allennlp/tests/fixtures/semantic_parsing/atis/serialization/vocabulary/rule_labels.txt

```diff
@@ -0,0 +1,205 @@
+agg -> [agg_func, "(", col_ref, ")"]
```

Comment:
How did you construct these fixtures?  I'd recommend adding a line to this script: https://github.com/allenai/allennlp/blob/8be358e9a495b0fad6202cca89ce29a5f07dc504/scripts/train_fixtures.py#L72

Then you comment out all of the models you don't want to retrain (all of them except the one you're working on), and run the script.

Reply:
Oh I see -- I trained it separately. I'll add to this script.